### ISSUE TITLE
comment-alt icon changed name and now only in pro

### ISSUE THREAD

**Issue by liorwohl** [FortAwesome/Font-Awesome#12829]
Hi
I was using icon regular fa-comment-alt in version 5.0.8 (1.0.4 in npm) that was looking like that:
![screen shot 2018-04-11 at 10 19 58](https://user-images.githubusercontent.com/10486742/38601949-f2c88896-3d71-11e8-85fd-7e90a5d3d4c1.png)

since 5.0.9 this icon is not available in free version, it became fa-comment-dots in pro version.

is it intended? should i worry that some icon in my app will disappear every time i update a minor version, or is it just a bug?

also, I see the icon solid comment-dots listed as free here:
https://fontawesome.com/icons?d=gallery&q=comment-&m=free
but when trying to use it i get nothing.

i am using the font icons, not the svg option.

thanks

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Thanks for reporting this.

Actually, comment-alt changed from 5.0.8 to 5.0.9

![image](https://user-images.githubusercontent.com/556268/38604117-b475d000-3d6f-11e8-99a8-141ac4b5797c.png)

It should not happen in a patch release because it is a potential breaking change

@talbs could you please take a look?

**Comment by talbs** [comment]
@liorwohl and @tagliala, thanks for bringing this to our attention. A breaking change wasn't on purpose and we'll be getting this fixed shortly. Hang tight!

**Comment by robmadole** [comment]
@liorwohl so this icon didn't vanish but the design of it changed, is that correct? So you consider this a breaking change because the design is now different?

**Comment by liorwohl** [comment]
yes, its a different icon. it doesn't fit what it was representing before.


also, no one is mentioning solid comment-dots which dont work at all

**Comment by talbs** [comment]
Hi, @liorwohl. Thanks for the extra info and helping us get this resolved. 

We circled our wagons as a team and our current plan is to keep the current version of `comment-alt` the way it is since it fits the new visual and naming convention standards we moved to with version 5. Its a visual alternative to the `comment` icon.

Now we totally recognize (thanks to you) that V4's `comment-alt` is a different icon than this one. It has always conveyed different information than `comment`. And to move it to our current standards and expand the content of comment/chat icons like we've done, we needed to rename that visual icon to `comment-dots`. 

We totally get how that messes with a look and feel of a project you're using FA on now. So we're going to do what we've done with other V4 icons that have visually changed but that our users have depended on in their existing projects, including:

 1. adding the regular style of `comment-dots` into Font Awesome Free for everyone to use
 2. mapping `fa fa-comment-alt` to translate to `far fa-comment-dots` for users of our Version 4 Shim file](https://fontawesome.com/how-to-use/upgrading-from-4#v4-shim).

We're getting these changes ready now. Hope that helps you keep using the icon you want while moving to Font Awesome 5's content and standards. 😄
